.github/workflows/haskell-ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       image: buildpack-deps:xenial
     services:
       postgres:
-        image: postgres:10
+        image: postgres:13
```

Comment:
This is not a minor documentation improvement at all...

Reply:
Yeah, but the tests weren't working, because these is a change in a dependency (hpqtypes) that doesn't work with psql 10

Reply:
Can you change it to 11? It's the lowest version that works.

Reply:
Ok, changed to 11.